### PR TITLE
Add Beta phase banner

### PR DESCRIPTION
**Pull request by Harriethw**
# Context

[Trello](https://trello.com/c/7Iu4Jm5f/220-add-banner-to-the-service-with-our-mailbox-in-it)

We need to show a [GDS phase banner](https://design-system.service.gov.uk/components/phase-banner/) on all pages of the app in order to capture feedback from users or receive emails from them. In the main sub-sections of the app we will need specific links to feedback forms as well as a link to email, but on the home page we just need copy to direct users to email us.

# Changes in this PR

This commit adds the component to the `header.njk` partial. We decided to sit it below the `mojPrimaryNavigation` because it did not look quite right sitting between the main header and the nav. Although the GDS guidance says to put it directly under the header, there's some discussion about [this being the right place](https://github.com/alphagov/govuk-design-system-backlog/issues/57) and this seems like the right thing for us.

### After
Screenshot 2023-06-20 at 11 31 58

Screenshot 2023-06-20 at 11 32 14

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: integration_tests/tests/login.cy.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ context('SignIn', () => {
 
   it('Token verification failure clears user session', () => {
     cy.signIn()
-    const indexPage = Page.verifyOnPage(PremisesListPage)
+    const indexPage = Page.verifyOnPage(DashboardPage)
     cy.task('stubVerifyToken', false)
```
Comment (richpjames) [inline integration_tests/tests/login.cy.ts:57]: Good catch

File: server/utils/phaseBannerUtils.test.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+describe('when there is a feedback link', () => {
+  const paths = [
+    ['/applications/example', APPLY_FEEDBACK_LINK],
+    ['/assessments/example', ASSESS_FEEDBACK_LINK],
+    ['/premises/example', MANAGE_FEEDBACK_LINK],
+    ['/placement-requests/example', MATCH_FEEDBACK_LINK],
+  ]
+
+  test.each(paths)('it returns correct feedback link for path', (path, link) => {
```
Comment (pezholio) [inline server/utils/phaseBannerUtils.test.ts:17]: Oooh, I didn't know about `test.each`!

Comment (richpjames) [inline server/utils/phaseBannerUtils.test.ts:17]: Did not know this syntax but I like it!

**Review by pezholio — APPROVED** [review]
Amazing! Looks really good - thank you!

File: server/utils/phaseBannerUtils.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+export const getContent = (currentUrl: string) => {
+  const feedbackLink = getFeedbackLink(currentUrl)
+
+  return feedbackLink
+    ? `This is a new service. <a class="govuk-link" href="${feedbackLink}">Give us your feedback</a> or <a class="govuk-link" href="mailto:APServiceSupport@digital.justice.gov.uk">email us</a> to report a bug`
+    : 'This is a new service. <a class="govuk-link" href="mailto:APServiceSupport@digital.justice.gov.uk">Email us</a> to report a bug'
+}
+
+const getFeedbackLink = (currentUrl: string) => {
+  if (currentUrl.startsWith('/applications')) {
```
Comment (Harriethw) [inline server/utils/phaseBannerUtils.ts:19]: i've just had a thought that I could use the `paths` here...

**Review by richpjames — APPROVED** [review]
LGTM, thanks!

**Pull request merged**